.github/ISSUE_TEMPLATE/frontend-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Frontend request
+about: Request for Frontend and Design work
+title: ''
+labels: Ui/UX
+assignees: joycehermosilla, verabehr
+
+---
+
+**Describe what you need**
+A clear and concise description of what you need.
```

Comment:
[Currently these two lines get rendered as one line](https://github.com/palomachain/paloma/blob/ffd3078339c78f0abcfb4e76d608060cae24c1e8/.github/ISSUE_TEMPLATE/frontend-request.md), I'm not sure that's intended.

Reply:
It's also not clear from context here what text is intended to be verbatim and what text is meant to be replaced. I think this section e.g. should be:

```
## Description

_A clear and concise description of what you need._
```

which renders as:

## Description

_A clear and concise description of what you need._